kumo-cli/src/main/java/com/kennycason/kumo/cli/CliParameters.java

```diff
@@ -78,10 +78,10 @@
     private int fontSizeMax = 40;
 
     @Parameter(names = "--font-weight", description = "A font weight. Default is Bold.", converter = FontWeightConverter.class)
-    private FontWeight fontWeight = FontWeight.BOLD;
+    private FontWeight fontWeight = FontWeight.PLAIN;
 
     @Parameter(names = "--font-type", description = "The name of the font to use. The system must have the font loaded already. Default is \"Comic Sans MS\".")
-    private String fontType = "Comic Sans MS";
+    private String fontType = "微软雅黑"; //"Comic Sans MS";
```

Comment:
I think we'll have to use a more universal default font, but allow user's to override if needed.